source/agora/node/Fee.d

```diff
@@ -0,0 +1,157 @@
+/*******************************************************************************
+
+    Contains the function to calculate the fees used to store the data
+
+    Copyright:
+        Copyright (c) 2019 BOS Platform Foundation Korea
+        All rights reserved.
+
+    License:
+        MIT License. See LICENSE for details.
+
+*******************************************************************************/
+
+module agora.node.Fee;
+
+import agora.common.Amount;
+
+import std.math;
+
+/*******************************************************************************
+
+    Calculates the fee of transaction data to store
+
+    Params:
+        data_size = The size of the data
+        factor = The factor to calculate for the fee of transaction data
```

Comment:
So when the factor is bigger, the fee is lower? That's what the tests are suggesting.

What is the "factor" here? What values can be accepted (which range)?

Reply:
https://github.com/bpfkorea/agora/pull/1321#issue-515139655
We can see from the graph.